FGEGraphics/UISystem/UIElement.cs

```diff
@@ -188,15 +184,7 @@ public bool Contains(int x, int y)
     /// <param name="x">The X position to check for.</param>
     /// <param name="y">The Y position to check for.</param>
     /// <returns>Whether the position is within any of the boundaries.</returns>
-    public bool SelfContains(int x, int y)
-    {
-        int lowX = LastAbsolutePosition.X;
-        int lowY = LastAbsolutePosition.Y;
-        int highX = lowX + LastAbsoluteSize.X;
-        int highY = lowY + LastAbsoluteSize.Y;
-        return x > lowX && x < highX
-            && y > lowY && y < highY;
-    }
+    public bool SelfContains(int x, int y) => x > X && x < X + Width && y > Y && y < Y + Height;
```

Comment:
This would be a preexisting issue, but, shouldn't this be `>=` rather than `>`? ie count the 1px boundary line as contained

Reply:
Interactions could overlap for two directly adjacent elements (a layout that is much easier to work with)

Reply:
Er... are those two directly overlapping elements* you mean?

Or is this an adjacency case of... the min is `==` but the max is not? ie `x >= X && x < X + Width`
Whichever case is visually correct, should also be considered the relevant truth for interaction processing